packages/devextreme/js/__internal/ui/m_map.ts

```diff
@@ -1,440 +1,368 @@
-import $ from '../core/renderer';
-import eventsEngine from '../events/core/events_engine';
-import { fromPromise } from '../core/utils/deferred';
-import registerComponent from '../core/component_registrator';
-import errors from './widget/ui.errors';
-import devices from '../core/devices';
-import Widget from './widget/ui.widget';
-import { titleize } from '../core/utils/inflector';
-import { each } from '../core/utils/iterator';
-import { extend } from '../core/utils/extend';
-import { wrapToArray } from '../core/utils/array';
-import { isNumeric } from '../core/utils/type';
-import { addNamespace } from '../events/utils/index';
-import pointerEvents from '../events/pointer';
-import { noop } from '../core/utils/common';
-
+import registerComponent from '@js/core/component_registrator';
```

Comment:
let's move `@ts/ui/m_map` to all other map files `@ts/ui/map/m_map`

Reply:
I will do this move in the next PR